order/views.py

```diff
@@ -1 +1,275 @@
+                "productSubName"  : order_product.product_stock.product.sub_name,
+                "productStockId"  : order_product.product_stock.id,
+                "productSize"     : order_product.product_stock.size,
+                "productPrice"    : order_product.product_stock.price,
+                "productStock"    : order_product.product_stock.stock,
+                "productImageUrl" : order_product.product_stock.product.image_set.get(is_main=True).image_url,
+                "productQuantity" : order_product.quantity
+                } for order_product in order.orderproductstock_set.filter(order=order)]
+
+            data = {
+                "orderNumber": order.order_number,
+                "carts": cart_product_list,
+            }
+
+            return JsonResponse({"data": data, "message": "SUCCESS"}, status=200)
+
+        except OrderStatus.DoesNotExist:
+            return JsonResponse({"message": "ORDER_DOES_NOT_EXIST"}, status=400)
+
+        except MultipleObjectsReturned:
+            return JsonResponse({"message": "MULTIPLE_OBJECTS_RETURNED"}, status=400)
+
+    @login_decorator
+    def post(self, request):
+        try:
+            data          = json.loads(request.body)
+            user          = request.user
+            product_id    = data['productId']
+            product_size  = data.get('productSize', None)
+            product_price = float(data['productPrice'])
+
+            if not Order.objects.filter(user=user, order_status=OrderStatus.objects.get(name='주문 전')).exists():
+                shipping_cost = DEFUALT_COST if product_price < MINIMUM_PRICE else DISCOUNTED_COST
+                
+                order_info    = Order.objects.create( 
+                    user            = user,
+                    order_number    = datetime.today().strftime("%Y%m%d") + str(randint(10000, 100000)),
+                    order_status    = OrderStatus.objects.get(name='주문 전'),
+                    sub_total_cost  = product_price,
+                    shipping_cost   = shipping_cost,
+                    total_cost      = product_price + shipping_cost
+                )
+            else:
+                order_info = Order.objects.get(user=user, order_status_id=1)
+                order_info.sub_total_cost = float(order_info.sub_total_cost) + product_price
+                order_info.shipping_cost  = DEFUALT_COST if order_info.sub_total_cost < MINIMUM_PRICE else DISCOUNTED_COST
+                order_info.total_cost     = float(order_info.sub_total_cost) + order_info.shipping_cost
+                order_info.save()
+
+            # order_product_stocks insert
+            added_product = ProductStock.objects.get(product_id=product_id, size=product_size)
+            OrderProductStock.objects.update_or_create(
+                order         = order_info,
+                product_stock = added_product,
+                defaults      = {
+                    "quantity" : 1
+                }
+            )
+
```

Comment:
```suggestion
                defaults      = {"quantity" : 1}
            )
        

```
키가 하나밖에 없기 때문에 한 줄로 간결하게 정리하시는게 좋을 것 같아요-!

Reply:
네 수정하였습니다~